src/parsing/css-translation-parsing.js

```diff
@@ -62,21 +39,17 @@
 
     switch (type) {
       case '3d' :
-        return translate3d(coords, remaining);
+        return internal.cssTranslationFromTranslate3d(coords, string, remaining);
```

Comment:
> This is incorrect anyway - this will put the entire input into the cssText of the CSSTranslation, instead of just the consumed part.

That is true. Forgot about that. I have a feeling we went through all of this when we did rotate and skew.